### ISSUE TITLE
Modpacks & Curseforge

### ISSUE THREAD

**Issue by sleepingdragoninn** [sleepingdragoninn/antique-atlas#80]
**TL;DR: You can redistribute this mod in modpacks. link the modrinth page in your description if possible - thanks!**

---

Hello! We have no plans to rehost this mod on curseforge, and don't want others to rehost it either.
If you're just a player, just grab the jar file or use a launcher that supports multiple mod platforms, like [Prism](https://prismlauncher.org/)!

For modpackers, Antique Atlas 4 is open source! feel free to download the jar and put it in your curseforge pack. 
If you're able, please include a line with a link anywhere in the description e.g. *This modpack includes [Antique Atlas 4](https://modrinth.com/mod/antique-atlas-4)!*

That's all!
